src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageDetailsTabViewModel.cs

```diff
@@ -73,8 +70,7 @@ public void Dispose()
                 return;
             }
             _disposed = true;
-            _readmeRenderingCancellationTokenSource.Cancel();
-            _readmeRenderingCancellationTokenSource.Dispose();
+            ReadmePreviewViewModel?.Dispose();
             DetailControlModel.PropertyChanged -= DetailControlModel_PropertyChanged;
```

Comment:
Would it be safer to dispose `ReadmePreviewViewModel` after unsubscribing this event handler?

Consider a race condition where the event handler fires after `ReadmePreviewViewModel` is disposed, `DetailControlModel_PropertyChanged` will access a disposed member [here](https://github.com/NuGet/NuGet.Client/pull/6266/files#diff-e34ff71887f227b90744191efd4f9982ff1a19f83551f0ff23676a29a3f95fd7R109).